# Do not link with libexif as the .so produced do not use its symbols
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -8,7 +8,6 @@ WriteMakefile(
     ($] >= 5.005 ?    ## Add these new keywords supported since 5.005
       (ABSTRACT_FROM => 'EXIF.pm', # retrieve abstract from module
        AUTHOR     => 'sergey s prozhogin <ccpro@rrelaxo.org.ru>') : ()),
-    'LIBS'		=> ['-lexif'], # e.g., '-lm'
     'DEFINE'		=> '', # e.g., '-DHAVE_SOMETHING'
 	# Insert -I. if you add *.h files later:
     'INC'		=> '', # e.g., '-I/usr/include/other'
